syntax = "proto3";
package dfi;

message DFIFlowBarrierRequest {
	string name = 1;
}
